packages/cli/src/index.ts

```diff
@@ -123,13 +136,74 @@ program
 program
     .command("get-group")
     .description("Get the data of a group from a supported network (Goerli or Arbitrum).")
-    .argument("<group-id>", "Identifier of the group.")
-    .option("-n, --network <network-name>", "Supported Ethereum network.", "goerli")
-    .option("--members", "Show group members.")
-    .option("--signals", "Show group signals.")
+    .argument("[group-id]", "Identifier of the group.")
+    .option("-n, --network <network-name>", "Supported Ethereum network.")
+    .option("-m, --members", "Show group members.")
+    .option("-s, --signals", "Show group signals.")
     .allowExcessArguments(false)
     .action(async (groupId, { network, members, signals }) => {
-        if (!["goerli", "arbitrum"].includes(network)) {
+        if (!network) {
+            const { selectedNetwork } = await inquirer.prompt({
+                name: "selectedNetwork",
+                type: "list",
+                message: "What is the network?",
```

Comment:
These prompt are created to show a list of options. I'd update the massage with something like "Select one of our supported networks:", like [`cz`](https://github.com/commitizen/cz-cli). "What is the network?" makes more sense when there are not options but only open answers.

"What is the group id?" -> "Select one of the following existing group ids" or something like that

How does it sound?

Reply:
Sounds good.